test/Microsoft.NET.Publish.Tests/Microsoft.NET.Publish.Tests.csproj

```diff
@@ -32,6 +32,8 @@
     </PackageReference>
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="$(RepositoryRootDirectory)src\Tasks\Microsoft.NET.Build.Tasks\PackageInfo.cs" Exclude="$(GlobalExclude)" />
```

Comment:
I don't think we should be doing this.

We have 2 types of tests:  unit and integration.

Unit tests can/should test each class in isolation and mock out things.  Put those tests in the UnitTests project under [https://github.com/dotnet/sdk/tree/master/src/Tasks/Microsoft.NET.Build.Tasks.UnitTests](https://github.com/dotnet/sdk/tree/master/src/Tasks/Microsoft.NET.Build.Tasks.UnitTests).

Integration tests should have 0 knowledge of the inner workings of the code.  This test project is an integration test.  It should only be doing things that users would see/interact with.